signup/signup.css

```diff
@@ -0,0 +1,142 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Poppins', sans-serif;
+    --product-name: #ff2020;
+    --primary-color: #e72744;
+    --secondary-color: #F05524;
+    --primary-btn: #6495ED;
+}
+
+
+/* nav bar */
+
+.landing-page-nav {
+    display: flex;
+    justify-content: space-between;
+    background-color: #f1f1f1;
+    padding: 10px;
+    top: 0;
+}
+
+.brand-name {
+    color: var(--primary-color);
+    font-size: 2.5rem;
+    font-weight: bold;
+}
+
+.brand-sub-title {
+    color: var(--secondary-color);
+    margin-top: -1rem;
+    margin-left: 1rem;
+}
+
+.nav-links {
+    padding-top: 20px;
+}
+
+.nav-links a {
+    color: black;
+    text-decoration: none;
+    padding-right: 15px;
+}
+
+.search-bar {
+    max-width: 800px;
+    width: 100%;
+}
+
+.search-bar-form {
+    display: flex;
+}
+
+.search-bar-input {
+    flex-grow: 1;
+    padding: 16px;
+    font-size: 16px;
+}
+
+.submit-btn {
+    padding: 15px;
+}
+
+.nav-icons {
+    display: flex;
+    justify-content: flex-end;
```

Comment:
You can make a class for this two properties and use it where you want. It will save the no of lines of code.

Reply:
hii manoj.will update the button with links .and for using px or rems i want to refactor that once i finish my P0 screen definitely will keep that in my mind .thanks for the preview